### ISSUE TITLE
Get file extension from clipper-upload

### ISSUE THREAD

**Issue by Domynos** [timtnleeProject/vuejs-clipper#37]
Hello,

Is there a way to get the original filename from clipper-upload component ? As you bind v-model with window.URL.createObjectURL(this.file), i can't have access to original file extension, right ?

Thanks for the plugin by the way.

**Comment by timtnleeProject** [comment]
Hi @Domynos ,
You can get the upload file object through the `clipper-upload` data `file`.

```html
<clipper-upload  $ref="upload" />
```

Then you can get the File object after either:
* Dom-string updated
* inside `clipper-upload` on `input` event callback. (ex: `<clipper-upload  $ref="upload" @input="cb" />`) 
* image using that dom-string is loaded. (ex: `<clipper-basic :src="src" @load="cb">`)

```javascript
this.$ref.upload.file // get the File object
```

**Comment by Domynos** [comment]
Hi @timtnleeProject 

I didn't know I could access to child component like that. Thank you, I learned something! 
Your syntax didn't work in my case maybe due to different vuejs versions ? 

I had to use :
`<clipper-upload ref="upload/> (instead of $ref)`

and access it by 

`this.$refs.upload.file`

Reference : https://vuejs.org/v2/guide/components-edge-cases.html#Accessing-Child-Component-Instances-amp-Child-Elements

Thank you for the quick response.
